modules/retail/pages/retail-requirements.adoc

```diff
@@ -63,14 +67,33 @@ xref:installation:general-requirements.adoc[].
 
 == Network Requirements
 
-* The {susemgr} Server requires a reliable and fast WAN connection.
-* The branch server requires a reliable WAN connection, to reach the {susemgr} Server.
-* If you are using a dedicated network, the branch server requires at least two network interfaces: one connected to the WAN with reachable {susemgr} server, and one connected to the internal branch LAN.
+* The {productname} Server requires a reliable and fast WAN connection.
+* The branch server requires a reliable WAN connection, to reach the {productname} Server.
+* If you are using a dedicated network, the branch server requires at least two network interfaces: one connected to the WAN with reachable {productname} server, and one connected to the internal branch LAN.
 * Terminals require a LAN connection to the branch server network.
 
 
 
-== POS Terminal UEFI Secure Booting Requirements
+== POS Terminal Requirements
+
+[cols="1,1", options="header"]
+.Hardware Requirements for Terminals
+|===
+| Hardware                 | Recommended
+| RAM:                     | Minimum 1 GB for hosts that need to run OS images built with Kiwi (PXE booted or not)
+| Disk Space:              | Disk space depends on size of the OS image
+|===
+
+// Shall we also mention Leap?
+For more information, see the documentation of the underlying system (in this case: {sles}{nbsp}15).
+
+For more information on {smr} POS terminals, see documentation on {productname} Salt clients (xref:client-configuration:client-config-overview.adoc[]).
+
+
+
+=== UEFI Secure Booting Requirements
+
+// Shall we also mention Leap?
```

Comment:
How binding word `supported` is here in docs? For SLE it is easy, as we provide support for SLE. But for Leap and Uyuni it is different picture.
UEFI Secure Boot should work on Leap, but I'm not sure about the supported statement.